sdk/evaluation/tests.yml

```diff
@@ -0,0 +1,15 @@
+trigger: none
+
+extends:
+    template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      ServiceDirectory: evaluation
+      MatrixReplace:
+        - TestSamples=.*/true
+      EnvVars:
+        AZURE_CLIENT_ID: $(TEMPLATE_CLIENT_ID)
+        AZURE_CLIENT_SECRET: $(TEMPLATE_CLIENT_SECRET)
+        AZURE_TENANT_ID: $(TEMPLATE_TENANT_ID)
+        AZURE_SUBSCRIPTION_ID: $(TEMPLATE_SUBSCRIPTION_ID)
+        TEST_MODE: 'RunLiveNoRecord' # use when allowing preparers to create the rgs for you
```

Comment:
The auto-filled prefix from `new-testresources` aligns with the service that you pass in. The `TEMPLATE` from source yml is from the fact that it's literally the service dir `TEMPLATE` as in `sdk/template/azure-template`.